### PR TITLE
Pre commit for ricecooker

### PR DESCRIPTION
**Pull request by AtKristijan**
Precommit for ricecooker

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
+        hooks:
+            -   id: black
+                types_or: [ python, pyi ]
+                exclude: '^.+?\.template$'
+    -   repo: https://github.com/pycqa/flake8
+        rev: 4.0.1
+        hooks:
+            -   id: flake8
+                exclude: '^.+?\.template$'
+    -   repo: git://github.com/pre-commit/pre-commit-hooks
```
Comment (rtibbles) [inline .pre-commit-config.yaml:13]:
```suggestion
    -   repo: https://github.com/pre-commit/pre-commit-hooks
```

A recent change in github security means that the unauthenticated `git://` protocol URLs will no longer work. I am making similar changes in our other repos: https://github.com/learningequality/kolibri/pull/8987

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -1,19 +1,31 @@
...
+            -   id: check-yaml
+                exclude: '^.+?\.template$'
+            -   id: check-added-large-files
+                exclude: '^.+?\.template$'
+            -   id: debug-statements
+                exclude: '^.+?\.template$'
+            -   id: end-of-file-fixer
+                exclude: '^.+?(\.json|\.template)$'
+
+    -   repo: git://github.com/asottile/reorder_python_imports
```
Comment (rtibbles) [inline .pre-commit-config.yaml:27]:
```suggestion
    -   repo: https://github.com/asottile/reorder_python_imports
```

**Review by rtibbles — COMMENTED** [review]
No concerns here - there's a bit more cleanup to do with the linting from the flake-8 checks though, they are somewhat extensive, so we could break them up into follow up issues.

File: .pre-commit-config.yaml (hunk 2 above)
Comment (AtKristijan) [reply .pre-commit-config.yaml]: changed

File: .pre-commit-config.yaml (hunk 1 above)
Comment (AtKristijan) [reply .pre-commit-config.yaml]: changed

**Comment by rtibbles** [comment]
Going to merge this so we can iterate on getting to passing tests again.

**Pull request merged**